README.md

```diff
@@ -0,0 +1,2124 @@
+
+A List usage example is shown below.
+
+```python
+my_list = client.get_list("my-distributed-list").blocking()
+
+# Add elements to the list
+my_list.add("item1")
+my_list.add("item2")
+
+# Remove the first element
+print("Removed: {}".format(my_list.remove_at(0))) # Outputs 'Removed: item1'
+
+# There is only one element left
+print("Current size is {}".format(my_list.size())) # Outputs 'Current size is 1'
+
+# Clear the list
+my_list.clear()
+```
+
+### 7.4.7. Using Ringbuffer
+
+Hazelcast Ringbuffer is a replicated but not partitioned data structure that stores its data in a ring-like structure. You can think of it as a circular array with a given capacity. Each Ringbuffer has a tail and a head. The tail is where the items are added and the head is where the items are overwritten or expired. You can reach each element in a Ringbuffer using a sequence ID, which is mapped to the elements between the head and tail (inclusive) of the Ringbuffer. For details, see the [Ringbuffer section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#ringbuffer) in the Hazelcast IMDG Reference Manual.
+
+A Ringbuffer usage example is shown below.
+
+```python
+ringbuffer = client.get_ringbuffer("my-distributed-ringbuffer").blocking()
+
+# Add two items into ring buffer
+ringbuffer.add(100)
+ringbuffer.add(200)
+
+# We start from the oldest item.
+# If you want to start from the next item, call ringbuffer.tail_sequence()+1
+sequence = ringbuffer.head_sequence()
+print(ringbuffer.read_one(sequence)) # Outputs '100'
+
+sequence += 1
+print(ringbuffer.read_one(sequence)) # Outputs '200'
+```
+
+### 7.4.8. Using Topic
+
+Hazelcast Topic is a distribution mechanism for publishing messages that are delivered to multiple subscribers. For details, see the [Topic section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#topic) in the Hazelcast IMDG Reference Manual.
+
+A Topic usage example is shown below.
+
+```python
+def print_on_message(topic_message):
+    print("Got message:", topic_message.message)
+
+# Get a Topic called "my-distributed-topic"
+topic = client.get_topic("my-distributed-topic")
+
+# Add a listener to the Topic
+topic.add_listener(print_on_message)
+
+# Publish a message to the Topic
+topic.publish("Hello to distributed world") # Outputs 'Got message: Hello to distributed world'
+```
+
+### 7.4.9. Using Lock
+
+Hazelcast Lock is a distributed lock implementation. You can synchronize Hazelcast members and clients using a `Lock`. For details, see the [Lock section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#lock) in the Hazelcast IMDG Reference Manual.
+
+A Lock usage example is shown below.
+
+```python
+lock = client.get_lock("my-distributed-lock").blocking()
+
+# Now lock the Lock and execute some guarded code.
+lock.lock()
+try:
+    # Do something here
+    pass
+finally:
+    lock.unlock()
+```
+
+### 7.4.10. Using Atomic Long
+
+Hazelcast Atom
```

Comment:
Here is the table listing the default logging levels that come with the `logging` module and numeric values that represent their severity: